api/buckets/service.go

```diff
@@ -59,6 +62,25 @@ var (
 
 	// errInvalidNodeType indicates a node with type other than raw of proto was encountered.
 	errInvalidNodeType = errors.New("invalid node type")
+
+	// baseArchiveStorageConfig is used as
+	baseArchiveStorageConfig = ffs.StorageConfig{
+		Hot: ffs.HotConfig{
+			Enabled: false,
+		},
+		Cold: ffs.ColdConfig{
+			Enabled: true,
+		},
+	}
```

Comment:
This is the base `StorageConfig` we use to store bucket data in Powergate. Biggest thing to note here is that hot storage is disabled. When a bucket is archived, we create a `ffs.FilConfig` from a `ArchiveConfig` and set `Cold.Filecoin` value with it.

Reply:
Correct. The `ArchiveConfig` that they can edit maps to `StorageConfig.Cold.FilConfig` so that is the only part of the `StorageConfig` they can effectively edit. They can't edit the `Hot` settings at all. `Hot` settings are exclusively controlled right here.